driver/gl/window.go

```diff
@@ -468,6 +468,11 @@ func (w *window) mouseClicked(viewport *glfw.Window, button glfw.MouseButton, ac
 	}
 	ev.Position = fyne.NewPos(w.mousePos.X-pad-x, w.mousePos.Y-pad-y)
 
+	//no need to verify position or focus, as focus is set properly at the end of func
```

Comment:
I disagree with this assertion - we don't know what sort of functionality may be in the FocusLost() or FocusGained() ... to call it unnecessarily may be a bad thing...